docs/guides/latest/volumesnapshot/deployment-backup.md

```diff
@@ -0,0 +1,262 @@
+persistentvolumeclaim/backup-pvc created
+```
+##### Deploy Workload:
+
+Now, deploy the following Deployment. Here, we have mounted the PVC to workload.
+
+Below, the YAML for the Deployment we are going to create.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-demo
+  name: stash-demo
+  namespace: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stash-demo
+  template:
+    metadata:
+      labels:
+        app: stash-demo
+      name: busybox
+    spec:
+      containers:
+      - args: ["touch source/data/sample-file.txt && sleep 3000"]
+        command: ["/bin/sh", "-c"]
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /source/data
+          name: source-data
+      restartPolicy: Always
+      volumes:
+      - name: source-data
+        persistentVolumeClaim:
+         claimName: backup-pvc
+```
+
+Let's create the deployment we have shown above.
+
+```console
+$ kubectl apply -f ./docs/examples/volume-snapshot/deployment.yaml
+deployment.apps/stash-demo created
+```
+
+Now, wait for deployment’s pod to go into the Running state.
+
+```console
+$ kubectl get pod -n demo 
+NAME                          READY   STATUS    RESTARTS   AGE
+stash-demo-5f67c6b4c4-sqgll   1/1     Running    0         17s
+```
+
+You can check that the `/source/data/` directory of this pod is populated with data from the `backup-pvc` PVC by using the following command,
+
+```console
+$ kubectl exec -n demo stash-demo-7ccd56bf5d-4x27d -- ls -R /source/data
+/source/data:
+lost+found
+sample-file.txt
+```
+
+##### Create BackupConfiguration:
+
+Now, we are going to create a `BackupConfiguration` crd for creating Volume Snapshot of `backup-pvc` PVC mounted to `/source/data` directory of `stash-demo` deployment. `BackupConfiguration` crd will create a `BackupSession` crd and start taking periodic backup of `/source/data` directory.
+
+Sample `BackupConfiguration` crd YAML are given below,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupConfiguration
+metadata:
+  name: deployments-volume-snapshot
+  namespace: demo
+spec:
+  schedule: "*\1 * * * *"
+  driver: VolumeSnapshotter
+  target:
+    ref:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: stash-demo
+    snapshotClassName: default-snapshot-class
+  retentionPolicy:
+    name: 'keep-last-5'
+    keepLast: 5
+    prune: true
+```
+
+Here,
+* `spec.schedule` is a [cron expression](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#schedule) that indicates that `BackupSession` will be created every 1 minute. Then `BackupSession` crd will start taking the periodic backup of the data.
+
+* `spec.driver` The driver indicates the name of the agent to use to back up the target. Supported values are `Restic`, `VolumeSnapshotter`. `VolumeSnapshotter` are used for creating/restoring volume snapshot.
```

Comment:
`spec.driver The driver indicates` -> `spec.driver indicates`

Reply:
Currently, Stash supports `Restic`, `VolumeSnapshotter` drivers. The `VolumeSnapshotter` is used to backup/restore PVC using VolumeSnapshot API.